packages/node_modules/samples/browser-auth-implicit/index.html

```diff
@@ -21,7 +21,7 @@ <h3 style="color:red" id="authentication-status">Not Authenticated</h3>
     </div>
 
     <!-- /bundle.js is the bundled Webex JavaScript SDK -->
```

Comment:
This comment no longer applies

Reply:
Do you want it renamed to `webex.min.js` or just get rid of the line entirely?

Reply:
You can remove all of these instances since `webex.min.js` is pretty self explanatory.